app/page.js

```diff
@@ -1,9 +1,11 @@
 "use client";
+
 import { useEffect, useState } from "react";
+
 import Movie from "./Movie";
 import Search from "./components/Search";
-import { getPopularMovies, searchMovie } from "./utils/api/moviesAPI";
 import Pagination from "./components/Pagination";
+import { getPopularMovies, searchMovie } from "./utils/api/moviesAPI";
```

Comment:
Add a blank line above to separate the components import group from the utils import group.